website/src/documentation/sdks/java-dependencies.md

```diff
@@ -26,336 +26,24 @@ behavior in the service. If you are using any of these packages in your code, be
 aware that some libraries are not forward-compatible and you may need to pin to
 the listed versions that will be in scope during execution.
 
-<p>To see the compile and runtime dependencies for your Beam SDK version, expand
-the relevant section below.</p>
+Compile and runtime dependencies for your Beam SDK version are listed in `BeamModulePlugin.groovy` in the Beam repository. To view them, perform the following steps:
 
-<details><summary markdown="span"><b>2.9.0</b></summary>
+1. Open `BeamModulePlugin.groovy`.
 
-<p>Beam SDK for Java 2.9.0 has the following compile and runtime dependencies.</p>
+    ```
+    https://raw.githubusercontent.com/apache/beam/v<version_number>/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy
```

Comment:
Nit: can we make the `<version_number>` into uppercase `<VERSION_NUMBER>` so it stands out even more? It makes it more evident that it needs to be replaced.

Reply:
I don't think the disambiguation is required here given the text and example below the code block, but since you're the second person to bring it up, I'll add the change in. :)